content/commands/ts.add/index.md

```diff
@@ -177,6 +178,33 @@ This override is effective only for this single command and does not set the tim
 This argument has no effect when a new time series is created by this command.
 </details>
 
+<details open><summary><code>IGNORE ignoreMaxTimediff ignoreMaxValDiff</code></summary> 
+
+is the policy for handling the insertion of a new sample. Both of the following conditions must be met.
+
+  - The timestamp of the new sample is less than the previous maximum timestamp (`ignoreMaxTimeDiff`).
```

Comment:
This is wrong now...
Before, the two conditions were described correctly, though separately, but now - they are wrong.
It is about the difference of the current timestamp from the previous timestamp `(timestamp - max_timestamp ≤ ignoreMaxTimeDiff)`, and about the absolute value difference of the current value from the value and the previous timestamp `abs(value - value_at_max_timestamp) ≤ ignoreMaxValDiff))`.

My latest comment was that we should also mention that this filter is applied only if you add data in-order `(timestamp ≥ max_timestamp)`.

If this is still unclear - let's discuss it.

Reply:
Per our Slack conversation, I've patched this up in the next push.